cms/djangoapps/contentstore/signals/handlers.py

```diff
@@ -155,6 +159,13 @@ def listen_for_course_publish(sender, course_key, **kwargs):  # pylint: disable=
     # Send to a signal for catalog info changes as well, but only once we know the transaction is committed.
     transaction.on_commit(lambda: emit_catalog_info_changed_signal(course_key))
 
+    if is_offline_mode_enabled(course_key):
+        requests.post(
```

Comment:
We need to discuss this with somebody from AXIM: @GlugovGrGlib @NiedielnitsevIvan  TBD